Autopilot/Test/Src/AttitudeManager/Test_OutputMixing.cpp

```diff
@@ -259,8 +259,8 @@ TEST(AttitudeManager_OutputMixing, CannotReturnOutOfRangeValuesWhenGivenValidInp
 
 	ASSERT_EQ(error.errorCode, 0);
 
-	ASSERT_TRUE( (mixedOutput[L_TAIL_OUT_CHANNEL] <= 100.0f ) && (mixedOutput[L_TAIL_OUT_CHANNEL] >= -100.0f ) );
-	ASSERT_TRUE( (mixedOutput[R_TAIL_OUT_CHANNEL] <= 100.0f ) && (mixedOutput[R_TAIL_OUT_CHANNEL] >= -100.0f ) );
+	ASSERT_TRUE( (mixedOutput[ELEVATOR_OUT_CHANNEL] <= 100.0f ) && (mixedOutput[ELEVATOR_OUT_CHANNEL] >= -100.0f ) );
```

Comment:
I would suggest using `EXPECT_TRUE` and `EXPECT_EQ`. That way if a test fails, you will know how many of the comparisons failed instead of only seeing the first one that flopped. 

Had I used this for the waypoint manager, it probs would've shortened debugging time by a bit.

Reply:
Good point. I'll add a task to the asana. I'm thinking we should go through all test files and make this change at some point.

Reply:
I can do this. I assume the only files that need to be change are in `Autopilot/Test/Src`